### PR TITLE
Exams by Sasha Vysokovsky IM-23

### PR DESCRIPTION
**Pull request by v-jester**

### CODE REVIEW

File: Tasks/day.js [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
-
-_parse_day_ = (s) => {
-  let i;
-  for (i = 0; i < D.length; i++) {
-    if (s.startsWith(D[i].toLowerCase())) {
-      return i + 1;
-    }
+'use strict';
+const WEEK_DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
```
Comment (illiakroshka) [inline Tasks/day.js:2]:
```suggestion
const WEEK_DAYS = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```
You should change the array elements to lowercase

File: Tasks/day.js [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
+const parseDay = (string) => {
+  let i = 0;
+  for (const day of WEEK_DAYS) {
+    if (string.startsWith(day.toLowerCase())) return i + 1;
```
Comment (illiakroshka) [inline Tasks/day.js:6]:
```suggestion
    if (string.startsWith(day) return i + 1;
```
You can delete the function 'toLowerCase'

File: Tasks/day.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
 'use strict';
 const WEEK_DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
```
Comment (nevrodegenerate) [inline Tasks/day.js:2]: Можна перейменувати назви днів на lower case, щоб не використовувати toLowerCase в 6 рядку

File: Tasks/sum.js [hunk 4]
```diff
@@ -1,20 +1,13 @@
...
-  let k = 5;
-  for (i of s) {
-    let t = typeof i;
-    if (t === 'number') {
-      if (sum.length > 0) {
-        const new_Sum = sum[sum.length - 1] + i;
-        sum.push(new_Sum);
-      }
+'use strict';
+function sumOfElement(...args) {
```
Comment (tshemsedinov) [inline Tasks/sum.js:2]: Prefer arrow functions

File: Tasks/sum.js [hunk 5]
```diff
@@ -1,20 +1,13 @@
...
+  let result = 0;
+  const values = args.flat(Infinity);
```
Comment (tshemsedinov) [inline Tasks/sum.js:4]: Processing nested arrays is out of scope but it's ok

File: .eslintrc.json [hunk 6]
```diff
@@ -1,268 +1,268 @@
...
-    ],
-    "keyword-spacing": [
-      "error",
-      {
-        "before": true,
-        "after": true,
-        "overrides": {
-          "function": {
-            "after": false
+    "env": {
```
Comment (tshemsedinov) [inline .eslintrc.json:2]: Do not change eslint config, it's a part of exam task

**Review by tshemsedinov — APPROVED** [review]

**Pull request closed without merging**